jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/install-common/install-common.sh

```diff
@@ -106,7 +106,7 @@ function configure_drivers(){
         if [ "${configMode}" = "xml" ]; then
           sed -i "s|<!-- ##DRIVERS## -->|${drivers}<!-- ##DRIVERS## -->|" $CONFIG_FILE
         elif [ "${configMode}" = "cli" ]; then
-          echo "${drivers}" > ${S2I_CLI_DRIVERS_FILE}
+          echo "${drivers}" >> ${S2I_CLI_DRIVERS_FILE}
```

Comment:
@yersan , @jbliznak  we can have an issue if the CONFIG_ADJUSTMENT_MODE == cli, the cli scipt will be executed multiple time, so the n-1 driver will be re-executed when adding the driver n. There is no clearing of S2I_CLI_DRIVERS_FILE.

Reply:
I agree. It seems we have to append the CLI operations here always but find a better place to invoke the exec_cli_scripts "${S2I_CLI_DRIVERS_FILE}" if we are in pure cli mode, probably after invoking the install.sh script